pkg/downloader/downloader.go

```diff
@@ -36,6 +36,13 @@ type Result struct {
 	ValidatedDigest bool
 }
 
+type CacheEntry struct {
+	ID       string        // sha256 of the Location
+	Location string        // source URL of the entry
+	Path     string        // Absolute path of the entry
+	Digest   digest.Digest // Checksum of the entry's data
```

Comment:
What data?

Reply:
This is about the cached data which is stored under the `data` file here: https://github.com/lima-vm/lima/blob/master/pkg/downloader/downloader.go#L168
Could you suggest a better comment maybe?

Reply:
Comment is improved